### PR TITLE
Added .keep for lib/iphone/rootless

### PR DESCRIPTION
**Pull request by MrGcGamer**
What does this implement/fix? Explain your changes.
---------------------------------------------------
This is just GitHub house-keeping `$THEOS/lib/iphone/rootless` already exists, after installation

Does this close any currently open issues?
------------------------------------------
Don't think so

Any relevant logs, error output, etc?
-------------------------------------

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Operating System:** …

**Platform:** …

**Target Platform:** …

**Toolchain Version:** …

**SDK Version:** …

### CODE REVIEW

**Comment by MrGcGamer** [comment]
included in #712

**Comment by MrGcGamer** [comment]
As #712 has been closed, without implementing this, I am reopening this PR.

File: lib/iphone/rootless/.keep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (leptos-null) [inline lib/iphone/rootless/.keep:1]: nit: this file is 1 byte (line break) instead of empty

**Review by leptos-null — CHANGES_REQUESTED** [review]
Small nit-pick for consistency with other `.keep` files in this repo.

This PR is important for `internal-library-stage_`/`internal-framework-stage_` to work correctly.

**Review by leptos-null — APPROVED** [review]
Thanks

**Review by uroboro — APPROVED** [review]

**Comment by leptos-null** [comment]
#801 now creates this directory as needed.
I'm merging this PR as well so that the directory is easy to access.
Any number of schemes may exist, so I'm planning to include (via `.keep` files) `lib/{target}/{scheme}` directories in this repositories that have a mirrored directory in https://github.com/theos/lib

**Pull request merged**